### PR TITLE
Optional nodePort selection for the service

### PR DESCRIPTION
**Pull request by Palollo**
Just adds the possibility to set the nodePort number.
Ref: https://kubernetes.io/docs/concepts/services-networking/service/#nodeport-custom-port

### CODE REVIEW

**Comment by Palollo** [comment]
The changes of this PR are tested in our cluster, setting the value of nodePortNumber and also leaving it empty.
The changes includes also the documentation update for the inclusion of the new properties.
Should I include the increment of chart version?

File: helm/oauth2-proxy/README.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ Parameter | Description | Default
 `service.clusterIP` | cluster ip address | `nil`
 `service.loadBalancerIP` | ip of load balancer | `nil`
 `service.loadBalancerSourceRanges` | allowed source ranges in load balancer | `nil`
+`service.nodePortNumber` | external port number for the service when service.type is `NodePort` | `nil`
```
Comment (desaintmartin) [inline helm/oauth2-proxy/README.md:162]: why `nodePortNumber` instead of `nodePort`?

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Could you bump minor version of Chart.yaml? This is new feature according to semver.

File: helm/oauth2-proxy/README.md (hunk 1 above)
Comment (Palollo) [reply helm/oauth2-proxy/README.md:162]: Because there is already the property `service.portNumber` defined 5 lines above.

Comment (desaintmartin) [reply helm/oauth2-proxy/README.md:162]: Oh, that's ugly, actually! I would prefer to get rid of this one later on and introduce sane naming now, I think `nodePort` is both cleaner and closer to what the Helm community does.

Comment (Palollo) [reply helm/oauth2-proxy/README.md]: ok, done

**Comment by desaintmartin** [comment]
Thanks!

**Pull request merged**